cabby/exceptions.py

```diff
@@ -18,7 +16,7 @@ class InvalidResponseError(ClientException):
 class UnsuccessfulStatusError(ClientException):
 
     def __init__(self, taxii_status, *args, **kwargs):
-        msg = "Server Error: {}".format(_status_to_message(self.raw))
+        msg = "Server Error: {}".format(_status_to_message(taxii_status))
```

Comment:
out of curiosity, this code base is supposed to work with pre `f"string"` python versions? (those are all end of life)

Reply:
Current code is still tested on and compatible with Python 2.7.

Reply:
I'd say it's not safe to drop Python 2.7 support yet. Cabby has quite a few 2.7 installation:

```bash
❯ dephell package downloads --filter=pythons cabby
WARNING cannot find config file 
[
  {
    "category": "27",
    "chart": "█▇▇▁▁▂▃ ▄▄▄▁▁▅▃ ▃▃▂▁▁▃▃ ▅▆▃▁▁█▅",
    "day": 18,
    "month": 461,
    "week": 107
  },
  {
    "category": "34",
    "chart": "_______ _______ ______█ _______",
    "day": 0,
    "month": 1,
    "week": 0
  },
  {
    "category": "35",
    "chart": "▁_▁__▅▅ ▃▃▁▅_▁▁ ▁▁█__▃▂ ▃▁▁__▁▁",
    "day": 1,
    "month": 52,
    "week": 7
  },
  {
    "category": "36",
    "chart": "▅▅▆▃_▇▇ ▅▅▆▂▂▆▄ ▆▅█▃▁▅▄ ▄▄▄▂▂▄▆",
    "day": 39,
    "month": 819,
    "week": 168
  },
  {
    "category": "37",
    "chart": "▄▂▂▂▃▂▄ ▅▂▇▂▁█▅ ▅▃▃_▂▅▂ ▄▃▆_▁▅▃",
    "day": 13,
    "month": 430,
    "week": 92
  },
  {
    "category": "38",
    "chart": "▅▃▁▂▂▅▅ ▅▆▃▃_▃▃ ▄▃▅▃_▅▃ ▅▅█▇▄▅▆",
    "day": 15,
    "month": 294,
    "week": 97
  },
  {
    "category": "39",
    "chart": "_____▄▄ ▄█_▄___ ______█ _____▄▄",
    "day": 1,
    "month": 10,
    "week": 2
  },
  {
    "category": "null",
    "chart": "▁▅_▁▁▁▄ █▁_____ ▁___▁▂▄ ▁_▅__▄▁",
    "day": 3,
    "month": 115,
    "week": 28
  }
]
```